src/typescript-service.ts

```diff
@@ -453,6 +454,70 @@ export class TypeScriptService {
             .startWith({ op: 'add', path: '', value: [] })
     }
 
+    /**
+     * The goto type definition request is sent from the client to the server to resolve the type
```

Comment:
Looks like the original function doc is still here, probably want to update that :)